thumbor/engines/pil.py

```diff
@@ -342,6 +352,30 @@ def convert_to_grayscale(self, update_image=True, with_alpha=True):
             self.image = image
         return image
 
+    def has_transparency(self):
+        has_transparency = 'A' in self.image.mode or 'transparency' in self.image.info
+        if has_transparency and numpy:
+            # If has transapency and have numpy
```

Comment:
If has **transparency**, not transapency. Anyway, the comment should read:

# If the image has alpha channel and numpy is installed
# we check for any pixels that are not opaque (255)